server-nodejs/conf_install.json

```diff
@@ -34,6 +34,13 @@
             "enable": true,
             "path" : "./extensions/user_setup.js"
         },
+        "jwt_delegate": {
+            "enable": true,
+            "path": "./extensions/jwt_delegate.js",
+            "conf": { 
+                "server": "https://syncplanet.io"
```

Comment:
why would we want every new damas-core instances pointing to this server by default? Also, is it important to have delegation enabled by default?

Reply:
Right, we don't want every new damas-core instances pointing on this server, excepting productions studios. 
It's not important to have delegation enabled. So, she must be disabled by default and about server URL I think there must not be. In the wiki, we can indicate URL for productions studios.
What do you think ?

Reply:
syncplanet.io is not part of the damas-core project (syncplanet.io use damas-core)